checks.d/jenkins.py

```diff
@@ -75,13 +105,19 @@ def _get_build_metadata(self, dir_name):
 
     def _get_build_results(self, instance_key, job_dir):
         job_name = os.path.basename(job_dir)
-
         try:
             dirs = glob(os.path.join(job_dir, 'builds', '*_*'))
+            # versions of Jenkins > 1.597 dropped the timestamp-named folders
+            if len(dirs) == 0:
+                dirs = glob(os.path.join(job_dir, 'builds', '[0-9]*'))
             if len(dirs) > 0:
-                dirs = sorted(dirs, reverse=True)
+                # versions of Jenkins > 1.597 need to be sorted by build number (integer)
+                try:
+                    dirs = sorted(dirs, key=lambda x: int(x.split('/')[-1]), reverse=True)
+                except:
```

Comment:
You should almost never use bare "except:" as it would also catch System exiting exceptions which you don't want.

You should use except Exception or even better catch the actual exceptions that can be caught here.